Cannot find coq-art on opam Coq repo

Wanted to install coq-art from opam as advised in README.md
> opam repo add coq-released https://coq.inria.fr/opam/released
> [...]
> opam install coq-coq-art
> [ERROR] No package named coq-coq-art found.

opam does not seem to be available
> opam search "coq*art"
> [no coq art here :,( ]

I cannot see it either in http://coq.io/opam/

--- a/README.md
+++ b/README.md
@@ -18,6 +18,7 @@
 [gitter-shield]: https://img.shields.io/badge/chat-on%20gitter-%23c1272d.svg
 [gitter-link]: https://gitter.im/coq-community/Lobby
 
+
 [doi-shield]: https://zenodo.org/badge/DOI/10.1007/978-3-662-07964-5.svg
 [doi-link]: https://doi.org/10.1007/978-3-662-07964-5
 
@@ -25,8 +26,6 @@ Coq'Art is the familiar name for the first book on the Coq proof assistant
 and its underlying theory, the Calculus of Inductive Constructions.
 This project contains the source of all examples and the solution to 170
 out of over 200 exercises from the book.
-
-
 
 ## Meta
 
@@ -39,29 +38,17 @@ out of over 200 exercises from the book.
 - License: [CeCILL-B](LICENSE)
 - Compatible Coq versions: 8.10
 - Additional Coq dependencies: none
+- Coq namespace: `coqart`
+- Related publication(s):
+  - [Interactive Theorem Proving and Program Development](http://www.labri.fr/perso/casteran/CoqArt/) doi:[10.1007/978-3-662-07964-5](https://doi.org/10.1007/978-3-662-07964-5)
 
-## Building and installation instructions
-
-The easiest way to install the latest released version of Coq'Art
-is via [OPAM](https://opam.ocaml.org/doc/Install.html):
-
-```shell
-opam repo add coq-released https://coq.inria.fr/opam/released
-opam install coq-coq-art
-```
-
-To instead build and install manually, do:
+## Building instructions
 
 ``` shell
 git clone https://github.com/coq-community/coq-art
 cd coq-art
 make   # or make -j <number-of-cores-on-your-machine>
-make install
 ```
-
-After installation, the included modules are available under
-the `coqart` namespace.
-
 
 ## Documentation
 

--- a/coq-coq-art.opam
+++ b/coq-coq-art.opam
@@ -1,5 +1,6 @@
 opam-version: "2.0"
 maintainer: "palmskog@gmail.com"
+version: "dev"
 
 homepage: "https://github.com/coq-community/coq-art"
 dev-repo: "git+https://github.com/coq-community/coq-art.git"
@@ -11,8 +12,7 @@ description: """
 Coq'Art is the familiar name for the first book on the Coq proof assistant
 and its underlying theory, the Calculus of Inductive Constructions.
 This project contains the source of all examples and the solution to 170
-out of over 200 exercises from the book.
-"""
+out of over 200 exercises from the book."""
 
 build: [make "-j%{jobs}%"]
 install: [make "install"]

--- a/meta.yml
+++ b/meta.yml
@@ -3,17 +3,21 @@ fullname: Coq'Art
 shortname: coq-art
 organization: coq-community
 community: true
+travis: true
+doi: 10.1007/978-3-662-07964-5
 
 synopsis: Coq sources and exercises from the Coq'Art book
 
-description: |
+description: |-
   Coq'Art is the familiar name for the first book on the Coq proof assistant
   and its underlying theory, the Calculus of Inductive Constructions.
   This project contains the source of all examples and the solution to 170
   out of over 200 exercises from the book.
 
-paper:
-  doi: 10.1007/978-3-662-07964-5
+publications:
+- pub_url: http://www.labri.fr/perso/casteran/CoqArt/
+  pub_title: Interactive Theorem Proving and Program Development
+  pub_doi: 10.1007/978-3-662-07964-5
 
 authors:
 - name: Yves Bertot
@@ -28,6 +32,8 @@ maintainers:
   nickname: Casteran
 
 opam-file-maintainer: palmskog@gmail.com
+
+opam-file-version: dev
 
 license:
   fullname: CeCILL-B
@@ -44,6 +50,15 @@ tested_coq_opam_versions:
 - version: '8.10'
 
 namespace: coqart
+
+build: |-
+ ## Building instructions
+
+ ``` shell
+ git clone https://github.com/coq-community/coq-art
+ cd coq-art
+ make   # or make -j <number-of-cores-on-your-machine>
+ ```
 
 keywords:
 - name: calculus of constructions
@@ -82,6 +97,7 @@ documentation: |-
   
   - [Tutorial on type classes](tutorial_type_classes)
   - [Tutorial on inductive and coinductive types](tutorial_inductive_co_inductive_types)
+  - [More exercises](more_exercises)
 
   [book-url]: http://www.labri.fr/perso/casteran/CoqArt/
   [publisher-url]: https://www.springer.com/gp/book/9783540208549

